storybook/src/components/FieldSet/FieldSet.stories.tsx

```diff
@@ -179,28 +182,40 @@ export const CheckboxGroup: Story = {
   ),
 }
 
-// export const CheckboxGroupWithError: Story = {
-//   args: {
-//     invalid: true,
-//     legend: 'Waar gaat uw melding over?',
-//   },
-//   render: (args) => (
-//     <FieldSet legend={args.legend} invalid={args.invalid}>
-//       {args.invalid && <ErrorMessage className="ams-mb--sm">Geef aan waar uw melding over gaat.</ErrorMessage>}
-//       <Column gap="extra-small">
-//         <Checkbox name="about" value="horeca" invalid={args.invalid} aria-required="true">
-//           Horecabedrijf
-//         </Checkbox>
-//         <Checkbox name="about" value="ander_bedrijf" invalid={args.invalid} aria-required="true">
-//           Ander soort bedrijf
-//         </Checkbox>
-//         <Checkbox name="about" value="evenement" invalid={args.invalid} aria-required="true">
-//           Evenement
-//         </Checkbox>
-//         <Checkbox name="about" value="anders" invalid={args.invalid} aria-required="true">
-//           Iets anders
-//         </Checkbox>
-//       </Column>
-//     </FieldSet>
-//   ),
-// }
+export const CheckboxGroupWithError: Story = {
+  args: {
+    invalid: true,
+    legend: 'Waar gaat uw melding over?',
+  },
+  render: (args) => (
+    <FieldSet
+      aria-labelledby="fieldset2 description7 error4"
```

Comment:
These numbers are a bit confusing: fieldset 2, description 7, error 4. This doesn’t help the explanations of how to populate the label.

I understand we need the `id`s to be unique for the entire docs page. So we can’t use field labels. Prefixing with the story name will make them overly long.

How about a letter per story, e.g. `a-fieldset`, `a-description`, `a-error`. And for examples with multiple inputs: `c-input-1`, `c-input-2`. A bit obscure as well maybe, but can’t think of anything better that is still compact.

Reply:
Also, making the error label dynamic helps prevent copy paste errors by developers.

```suggestion
      aria-labelledby=`fieldset2 description7 {invalid ? `error4` : undefined}`
```

Reply:
@VincentSmedinga, don't you want to send an empty string rather than `undefined` to a string list?

Reply:
> Also, making the error label dynamic helps prevent copy paste errors by developers.

And it doesn't lead to code that is flagged by the WAVE a11y checker, so I have to change this anyway 😁.

A letter per story is a good idea imo. I'd prefer a syntax like this though, which reads a little easier imo:

`fieldset-a`, `description-a`, `input-b1`, `input-b2`.

Reply:
It is easier to read. Technically, the letter represents the story and you want to express the hierarchy of parts (like we do with tokens). You could get in trouble with multiple examples that have multiple field sets that have multiple inputs, but fine to have the letter later and see how it goes.